examples/index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>

```

Comment:
Do you have something like this for the main kpt site?
Ideally we should create a folder called `site` and store it there if possible.

Reply:
We could do that in the long term, but that would break running it locally without something like nginx, since the site expects its contents like examples to be in child directories.

Reply:
> that would break running it locally without something like nginx

How do we bring up the catalog site locally? Why nginx can help here?

Reply:
Currently, you can do something like `cd examples/ && python -m SimpleHTTPServer` to start running. If the index and catalog README are in a directory outside of `examples`, we'd need something rerouting `/` to `../examples/`